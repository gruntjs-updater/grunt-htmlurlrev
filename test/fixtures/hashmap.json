{
  "img/eggs.png": "123abce1"
}
