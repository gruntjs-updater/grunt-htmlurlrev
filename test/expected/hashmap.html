<html>
  <head>
    <title>default</title>
  </head>
  <body>
    <img src="../img/123abce1.eggs.png" />
  </body>
</html>